shell/platform/darwin/ios/framework/Source/accessibility_bridge.h

```diff
@@ -98,10 +97,8 @@ class AccessibilityBridge final : public AccessibilityBridgeIos {
   // (i.e. the status bar or keyboard)
   int32_t last_focused_semantics_object_id_;
 
-  // TODO(cbracken): https://github.com/flutter/flutter/issues/137801
-  // Eliminate use of fml::scoped_* wrappers here.
-  fml::scoped_nsobject<NSMutableDictionary<NSNumber*, SemanticsObject*>> objects_;
-  fml::scoped_nsprotocol<FlutterBasicMessageChannel*> accessibility_channel_;
+  NSMutableDictionary<NSNumber*, SemanticsObject*>* objects_;
+  FlutterBasicMessageChannel* accessibility_channel_;
```

Comment:
I guess even if they are fully Objective-C isa objects we don't use Objective-C naming (`objects` and `accessibilityChannel`)?

Reply:
In C++ contexts (C++ class fields) the style guide says we have to use C++ naming, which really hits the code with an ugly stick :(

Reply:
If the naming of ivars and variables depended on the thing being pointed to rather than the context, things would be much, much uglier.

I guarantee nobody would be happier if individual lines of Obj-C++ code had mixes of different variable naming styles depending on the underlying types of the things the variables are referring to.

Reply:
Agreed, but it's still unfortunate. I'd much prefer we reduced the number of C++ classes with Obj-C++ implementations and used straight Obj-C[++] classes where possible.